config/consts_unix.go

```diff
@@ -0,0 +1,8 @@
+//go:build !windows
+
+package config
+
+const (
+	DefaultPython = "python3"
+	VenvPython    = "bin/python"
```

Comment:
These should not be a compile-time constant, since when we are building, it's always under unix environment. Maybe we should use something like `runtime.GOOS == "windows"` instead.